lib/ask/runtime/session.ex

```diff
@@ -637,9 +642,12 @@ defmodule Ask.Runtime.Session do
   end
 
   defp assign_bucket_if_one_match([bucket], respondent, session) do
-    respondent = respondent |> Respondent.changeset(%{quota_bucket_id: bucket.id}) |> Repo.update!
+    respondent =
+    respondent |> Respondent.changeset(%{quota_bucket_id: bucket.id}) |> Repo.update!()
```

Comment:
Yes, it was just my formatter that changed that. I applied it for the whole function.

I have installed these two extensions: [vscode-elixir-formatter](https://github.com/jakesorce/vscode-elixir-formatter) and [vscode-elixir-ls](https://github.com/JakeBecker/vscode-elixir-ls)

I'm not sure, but I think both of them use just the standard `mix format` command. I've tested both and they do the same for this line.

Specifically between these two options `Repo.update!` and `Repo.update!()` I think I would choose the one with the parenthesis, just because it's a function call, but... 🤷‍♂️

Would you like I revert it?

Reply:
No, if it's the formatter work, just leave it like that